test/test_data_dict.yaml

```diff
@@ -0,0 +1,6 @@
+unsorted_list:
```

Comment:
These names seemed to be mis-matched. I switched two elements in this list but forgot about the other list.

This would be noise to fix in JSON in this PR.